tests/Microsoft.Bot.Builder.Dialogs.Tests/ChoicePromptTests.cs

```diff
@@ -27,21 +25,53 @@ public class ChoicePromptTests
             new Choice { Value = "blue" },
         };
 
-        [TestMethod]
-        [ExpectedException(typeof(ArgumentNullException))]
+        /// <summary>
+        /// Generates an Enumerable of variations on all supported locales.
+        /// </summary>
+        /// <returns>An iterable collection of objects.</returns>
+        public static IEnumerable<object[]> GetLocaleVariationTest()
```

Comment:
Why did you change this method from private to public?

Reply:
It recommends using it as public otherwise it shows an xUnit error.
![image](https://user-images.githubusercontent.com/62260472/92516822-1da2b600-f1ec-11ea-95c4-8508e02aaf8d.png)